data.json

```diff
@@ -1348,5 +1348,11 @@
     "githubUsername": "shreyaansjain06",
     "imageUrl": "https://user-images.githubusercontent.com/51391473/134031514-b0bbff29-3974-4f05-a2c6-a5c163e90216.png",
     "issueNumber": 726
+  },
+  {
+    "name": "Rehan Punjwani",
+    "githubUsername": "rehanpunjwani",
+    "imageUrl": "https://user-images.githubusercontent.com/49568477/134772561-9e56321a-e6fa-4e85-89a7-52adcb447ffd.png",
```

Comment:
I would recommend more of an image like this.

![image](https://user-images.githubusercontent.com/51391473/134791010-7e18ffeb-f5f1-4b0f-ac41-4c4bc9006485.png)

```suggestion
    "imageUrl": "https://user-images.githubusercontent.com/51391473/134791010-7e18ffeb-f5f1-4b0f-ac41-4c4bc9006485.png",
```